### PR TITLE
Integrated serverless SLM model deployments in prem-utils

### PR DESCRIPTION
**Pull request by Anindyadeep**
This PR:
- solves issue #30 
- Models include: `tinyllama`, `stable_lm2`, `mamba`, `phi1.5` and `phi2`
- Supports both normal completion and streaming

### CODE REVIEW

File: prem_utils/default_urls.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+DEFAULT_PREM_URLS = {
```
Comment (filopedraz) [inline prem_utils/default_urls.py:1]: I would move this variable in the `prem.py` file at the class level.

`self.url_mappings = {}`  in the PremConnector constructor.

**Review by filopedraz — CHANGES_REQUESTED** [review]

File: prem_utils/default_urls.py (hunk 1 above)
Comment (Anindyadeep) [reply prem_utils/default_urls.py:1]: Ahh okay makes sense

**Comment by filopedraz** [comment]
@Anindyadeep you should not remove the ones we are serving with Prem. We should just have multiple versions of the same models.

e.g., `phi2`and `phi2-modal` :)

**Comment by Anindyadeep** [comment]
> @Anindyadeep you should not remove the ones we are serving with Prem. We should just have multiple versions of the same models.
> 
> e.g., `phi2`and `phi2-modal` :)

I see, makes sense, and done :)

**Pull request merged**